content/intro-to-storybook/react/en/data.md

```diff
@@ -77,6 +77,27 @@ const defaultTasks = [
 export default createStore(reducer, { tasks: defaultTasks });
 ```
 
+After that, we need to update our `src/App.js` to be able to connect our React components to the Redux store. The <Provider /> makes the Redux store available to any nested components that have been wrapped in the connect() function.
```

Comment:
@cdelacombaz we probably could rephrase this and avoid having a "carbon copy" (pardon the bad pun) of the Redux documentation for the provider component. I think that we could use the phrasing that is used in the [Vue edition](https://www.learnstorybook.com/intro-to-storybook/vue/en/data/). Turning this into:
"In our top-level app component (src/App.js) we can wire the store into our component hierarchy fairly easily:"

Also @domyen and @domyen are you ok with the phrasing

Reply:
Thanks for the feedback. I have updated the text as suggested in case you still want to merge it :)

Reply:
@cdelacombaz just one minor tweak that i forgot to mention and this should be good to go. Instead of:

```javascript
import React, { Component } from 'react';
import { Provider } from 'react-redux';
import store from './lib/redux';
import TaskList from './components/TaskList';
class App extends Component {
  render() {
    return (
      <Provider store={store}>
        <TaskList />
      </Provider>
    );
  }
}
export default App;
```

We can simplify this code snippet and maintain the one used by default with CRA, turning it into:

```javascript
import React from 'react';
import { Provider } from 'react-redux';
import store from './lib/redux';
import TaskList from './components/TaskList';

function App() {
  return (
    <Provider store={store}>
       <TaskList />
    </Provider>
  );
}

export default App;
```

As it will tie into the next chapter of the tutorial, allowing code consistency.